DocumentedExamples/regional-mom6-forced-by-access-om2.ipynb

```diff
@@ -0,0 +1,699 @@
+{
```

Comment:
<p>why was this commented out @luweiyang? is the dask client create issues?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>No, I commented them out when I was debugging because at some stage I did get some client-related errors. But it's no longer an issue. I will uncomment them. </p>

Reply:
<p>OK, do we need to add this back or not?</p>